content/en/docs/concepts/services-networking/service.md

```diff
@@ -534,6 +534,25 @@ to just expose one or more nodes' IPs directly.
 Note that this Service is visible as `<NodeIP>:spec.ports[*].nodePort`
 and `.spec.clusterIP:spec.ports[*].port`. (If the `--nodeport-addresses` flag in kube-proxy is set, <NodeIP> would be filtered NodeIP(s).)
 
+For example:
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: my-service
+spec:
+  type: NodePort
+  selector:
+    app: MyApp
+  ports:
+    - port: 80
+      targetPort: 80
+      # By default and for convenience, the `targetPort` is set to the same value as the `port` field.
+      nodePort: 30007
+      # Optional field
+      # By default and for convenience, the Kubernetes control plane will allocate a port from a range (default: 30000-32767)
```

Comment:
I'd suggest we put comments before the related YAML key. It is a convention as I see it.

Reply:
I refer to https://kubernetes.io/docs/concepts/services-networking/service/#other-clb-annotations-on-tencent-kubernetes-engine-tke.

Let me follow this style

Reply:
updated thx again